docs/source/development/traffic_ops_api/v12/deliveryservice.rst

```diff
@@ -770,29 +770,33 @@ SSL Keys
+  | ``crt``             | string | base64 encoded crt file for delivery service                                                                                            |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``csr``             | string | base64 encoded csr file for delivery service                                                                                            |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``key``             | string | base64 encoded private key file for delivery service                                                                                    |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``cdn``             | string | The CDN entered by the user when generating certs.                                                                                      |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``deliveryservice`` | string | The delivery service entered by the user when generating certs.                                                                         |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``businessUnit``    | string | The business unit entered by the user when generating certs.  Field is optional and if not provided by the user will not be in response |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``city``            | string | The city entered by the user when generating certs.  Field is optional and if not provided by the user will not be in response          |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``organization``    | string | The organization entered by the user when generating certs.  Field is optional and if not provided by the user will not be in response  |
+  +---------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------+
+  | ``hostname``        | string | The hostname entered by the user when generating certs.  Field is optional and if not provided by the user will not be in response      |
```

Comment:
The hostname generated by Traffic Ops that is used as the common name when generating the certificate.  This will be a FQDN for DNS delivery services and a wildcard URL for HTTP delivery services.  This field is not user entered.